pro/src/screens/Bookings/BookingsRecapTable/__specs__/BookingsRecapTable.spec.tsx

```diff
@@ -264,11 +265,12 @@ describe('components | BookingsRecapTable', () => {
     // Then
     // 1 line = 6 cells
     const cells = screen.getAllByRole('columnheader')
-    expect(cells).toHaveLength(5)
-    expect(cells[0]).toHaveTextContent('Nom de l’offre')
-    expect(cells[1]).toHaveTextContent('Établissement')
-    expect(cells[2]).toHaveTextContent('Places et prix')
-    expect(cells[3]).toHaveTextContent('Statut')
-    expect(cells[4]).toHaveTextContent('')
+    expect(cells).toHaveLength(6)
+    expect(cells[0]).toHaveTextContent('N° de réservation')
+    expect(cells[1]).toHaveTextContent('Nom de l’offre')
+    expect(cells[2]).toHaveTextContent('Établissement')
+    expect(cells[3]).toHaveTextContent('Places et prix')
+    expect(cells[4]).toHaveTextContent('Statut')
+    expect(cells[5]).toHaveTextContent('')
```

Comment:
Je sais que ce n'est pas toi qui l'a modifié, c'est juste un comment pour que tu transmettes à Charlotte quand vous bosserez dessus: il ne faudrait pas de header vide pour l'accessibilité. Si niveau design elle tient vraiment à ce que ce soit vide, alors il faudrait qu'on l'affiche mais qu'il soit caché visuellement

Reply:
C'est bien noté je transmets à Charlotte, merci 👍